### PR TITLE
Add Webapi.Canvas.toBlob & Webapi.Canvas.toDataUrl methods 

### LINKED ISSUE #109: HTML canvas element `toBlob` method is missing
Hi, 

Thanks for this awesome bindings!

I was looking to convert a canvas as a File to upload it, and it seems that the method required (toBlob) is missing.

https://developer.mozilla.org/en-US/docs/Web/API/HTMLCanvasElement/toBlob

### PR DESCRIPTION
**Pull request by MoOx**
(with alternatives for options)

Closes #109

### CODE REVIEW

**Review by TheSpyder — APPROVED** [review]
The "everything included" methods are a bit wordy but I can't come up with anything better 😂 

Sorry it took so long to get to this!

**Comment by TheSpyder** [comment]
`The module or file Blob can't be found`

I really need to add CI to this project to check when PRs haven't been compiled before I merge them 😂

**Comment by TheSpyder** [comment]
Published as 0.7.0

**Comment by alex35mil** [comment]
Why does `toDataUrl` (and similar methods) return `unit` instead of `string`?

**Comment by TheSpyder** [comment]
Ah. I only confirmed the `toBlob` signature in the original PR, I didn't check both. oops.

It's an easy fix, or would you like to make the PR for it?
